docs/en/observability/user-experience.asciidoc

```diff
@@ -87,41 +88,53 @@ image::images/user-exp-metrics.png[User experience metrics]
 .Metric reference
 ====
 First contentful paint::
-The timestamp when the browser begins rendering content (images, text, etc.) from the DOM.
-This is when the user can tell that the page is loading.
-footnote:[Source and more info: https://developer.mozilla.org/en-US/docs/Glossary/First_contentful_paint[developer.mozilla.org]]
+Focusses on the initial rendering and measures the time from when the page starts loading to when
+any part of the page's content is displayed on the screen.
+The agent uses the https://www.w3.org/TR/paint-timing/#first-contentful-paint[Paint timing API] available
+in the browser to capture the timing information.
+FCP is captured as transaction mark for `page-load` transaction for all chromium-based browsers
```

Comment:
I would remove the RUM agent specific details from this page, Might confuse users. thoughts?

Reply:
Makes sense